src/hooks/useQuizEvents/index.ts

```diff
@@ -23,6 +23,7 @@ const useQuizEvents: UseQuizEvents = (quizOptions, cioClient, quizState) => {
     dispatchLocalState,
     dispatchApiState,
     hasQuizStoredState,
+    isQuizCompleted,
```

Comment:
If you notice here we're just passing `isQuizCompleted` to `useQuizEvents` and then return it as is. I think you did this following what is done for `hasQuizStoredState` But I think it's not necessary for it as well

Can we return those values from useQuizLocalState directly without passing them to useQuizEvents?